tests/flow/test_persistency.py

```diff
@@ -56,8 +62,12 @@ def populate_graph(self, graph_name):
             # Create index.
             actual_result = redis_con.execute_command(
                 "GRAPH.QUERY", graph_name, "CREATE INDEX ON :person(name)")
+            actual_result = redis_con.execute_command(
+                "GRAPH.QUERY", graph_name, "CREATE INDEX ON :person(height)")
```

Comment:
combine the two into a single call: `CREATE INDEX ON :Person(name, height)`
See: [issue](https://github.com/RedisGraph/RedisGraph/issues/1781)